src/pages/components/PostList.tsx

```diff
@@ -0,0 +1,139 @@
+import Image from 'next/image'
+import styles from '@/src/styles/PostList.module.css'
+import Link from 'next/link'
+import { useEffect, useState } from 'react'
+import { getDeviceType, debounce } from '@/src/utils'
+import axiosPanda from '../api/axiosPanda'
+import Button from './Button'
+
+const sortQuery = {
+  최신순: 'recent',
+  인기순: 'like',
+}
+
+const sortOrderList = ['최신순', '인기순']

```

Comment:
상수로 선언을 잘해주셨어요!

type safe하게 한다면 하나만 추가하면 될것같습니다!
```suggestion
const sortQuery = {
  최신순: 'recent',
  인기순: 'like',
} as const

const sortOrderList: Array<keyof typeof sortQuery> = ['최신순', '인기순']
```